manifests/common/connection-types/oci-compliant-registry-v1.yaml

```diff
@@ -0,0 +1,17 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: oci-compliant-registry-v1
+  namespace: opendatahub
+  creationTimestamp: null
+  labels:
+    opendatahub.io/connection-type: 'true'
+    opendatahub.io/dashboard: 'true'
+  annotations:
+    opendatahub.io/enabled: 'true'
+    opendatahub.io/username: cluster-admin
```

Comment:
I don't believe we should include the namespace, creationTimestamp or username annotation.